src/testshade/optixgridrender.cpp

```diff
@@ -169,8 +169,11 @@ OptixGridRenderer::make_optix_materials ()
         shadingsys->optimize_group (groupref.get(), nullptr);
 
         if (!shadingsys->find_symbol (*groupref.get(), ustring(outputs[0]))) {
-            errhandler().warning ("Requested output '%s', which wasn't found",
-                                  outputs[0]);
+            // Asume a 1 x 1 image maybe testing things other than Cout 
+            if (m_xres > 1 || m_yres > 1) {
```

Comment:
What's going on here?

Reply:
That message will breaking tests that don't use Cout by inserting unexpected output.

Reply:
Yeah, but I'm not sure what that has to do with the resolution of the image

Reply:
At least a FIXME to explain why this is like it is and how it needs to be fixed eventually?